docs/auth/social.mdx

```diff
@@ -202,7 +202,81 @@ services (such as Facebook, Twitter, Google etc) must also have an Apple Sign-In
 To integrate Apple Sign-In on your iOS application, you will need to install a 3rd party library to authenticate with Apple.
 Once authentication is successful, a Firebase credential can then be used to sign the user into Firebase with their Apple account
 
-> This example is still TODO. PRs are welcome to provide an example (there's an 'Edit this page' link at the bottom of this page). 
+The example below is based on [sign_in_with_apple](https://pub.dev/packages/sign_in_with_apple) Flutter package.
+
+```dart
+import 'dart:math';
```

Comment:
For completeness, the user will also have to import these packages:

```dart
import 'dart:convert';
import 'dart:io';
```

Reply:
Thanks, I missed it when copying the example from my project. Updated.